client/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
-import App from "./App";
+import App from "./components/App";
 // import * as serviceWorker from './serviceWorker';
 
```

Comment:
Remove comments

Reply:
ok will do .. am following a school of thought that allows the react + ruby to sit in the same app folder

Reply:
Do you have the instance of the ruby hosted? This will allow me to test the react repo independently?